tests/units/test_var.py

```diff
@@ -1893,3 +1894,27 @@ def test_var_data_hooks():
 def test_var_data_with_hooks_value():
     var_data = VarData(hooks={"what": VarData(hooks={"whot": VarData(hooks="whott")})})
     assert var_data == VarData(hooks=["what", "whot", "whott"])
+
+
+def test_str_var_in_components(mocker):
+    class StateWithVar(rx.State):
+        field: int = 1
+
+    mocker.patch(
+        "reflex.components.base.bare.get_performance_mode",
+        lambda: PerformanceMode.RAISE,
```

Comment:
We're using `return_value` in other place where we patch function, so we should keep things consistent I think.
```suggestion
        return_value=PerformanceMode.RAISE,
```